frontend/src/app/view-claims/view-claims.component.html

```diff
@@ -4,14 +4,14 @@
     <!--STATUS # 0-->
   <ly-tab>
     <button ly-tab-label>Awaiting Settlement Option</button>
-    <ly-grid *ngIf="sortClaims(0).length > 0; else noClaims" container [spacing]="16">
+    <ly-grid *ngIf="sortClaims(0).length > 0; else hasClaims" container [spacing]="16">
```

Comment:
This looks like you check if there is any claims, and if not... go to `hasClaims`? What is this doing?